src/styles/index.scss

```diff
@@ -1,3 +1,71 @@
+$week: (
+  'mon': 0,
+  'tue': 1,
+  'wed': 2,
+  'thu': 3,
+  'fri': 4,
+  'sat': 5,
+  'sun': 6,
+);
+
 body {
   margin: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+}
+
+.calendar {
+  box-sizing: border-box;
+  width: 726px;
+  display: flex;
+  flex-wrap: wrap;
+  gap: 1px;
+  padding: 10px;
+
+  &__day {
+    display: flex;
+    position: relative;
+    width: 100px;
```

Comment:
let's move all metrics (cell size, gap, etc.) to variables and make all calculations based on these variables (such as calendar width or margins)